pages/index.js

```diff
@@ -20,12 +16,12 @@ export default function ShoppingItemsList({
 
   const [editedItem, setEditedItem] = useState({});
 
-  // const [isPurchased, setIsPurchased] = useState("");
-
   function handleChangeMode(mode) {
     setMode(mode);
   }
 
+  console.log("mode: ", mode);
```

Comment:
Please remove the console.log() as this is only needed for development. 🙂 
```suggestion

```

Reply:
Done.